.github/workflows/maven.yml

```diff
@@ -27,8 +7,8 @@ on:
     branches: [ main ]
 
 jobs:
-  build:
-    name: "JDK 21 Eclipse Temurin"
+  linux-build:
+    name: "JDK 21 Eclipse Temurin Linux"
     runs-on: ubuntu-latest
     container: "maven:3.9.6-eclipse-temurin-21"
```

Comment:
Why would a custom container be needed? Isn't the default option enough?
Also, removing this container and using a run matrix will remove duplication for linux and windows runs